instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -50,17 +58,33 @@ class _DjangoMiddleware(MiddlewareMixin):
     else:
         _excluded_urls = ExcludeList(_excluded_urls)
 
-    def process_view(
-        self, request, view_func, view_args, view_kwargs
-    ):  # pylint: disable=unused-argument
+    @staticmethod
+    def _get_span_name(request):
+        try:
+            if getattr(request, "resolver_match"):
+                match = request.resolver_match
+            else:
+                match = resolve(request.get_full_path())
+
+            if hasattr(match, "route"):
+                return match.route
+
+            # Instead of using `view_name`, better to use `_func_path` as some applications can use similar
+            # view names in different modules
+            if hasattr(match, "_func_name"):
+                return match._func_name  # pylint: disable=protected-access
+
+            # Fallback for safety as `_func_name` private field
+            return match.view_name
```

Comment:
Is this the same as `_func_name`?

Reply:
no, this is just short view function name, without module, for example:
_func_name = api.v1.cities.main
view_name = main